### PR TITLE
Update modulesync_config [a3fe424]

### PR DESCRIPTION
**Pull request by DavidS**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,9 +19,12 @@ matrix:
     script: bundle exec rake beaker
     services: docker
     sudo: required
+  - rvm: 2.3.1
+    bundler_args: --without system_tests
+    env: PUPPET_GEM_VERSION="~> 4.0"
   - rvm: 2.1.9
     bundler_args: --without system_tests
-    env: PUPPET_GEM_VERSION="~> 4.0" STRICT_VARIABLES="yes"
```
Comment (ghoneycutt) [inline .travis.yml]: Why stop supporting puppet v4 with strict variables?

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,508 @@
+require: rubocop-rspec
```
Comment (ghoneycutt) [inline .rubocop.yml]: This seems to have diverged quite a bit from the one discussed and implemented[1] in the module tool. Are there any docs or discussions on puppet-users that go over this?

It seems that it is explicitly enabling cops as opposed to enabling all of them and then ignoring some. Is that true, is it just being really explicit or are these extras not enabled by default?

[1] - https://github.com/puppetlabs/puppet/blob/master/lib/puppet/module_tool/skeleton/templates/generator/.rubocop.yml

File: .travis.yml (hunk 1 above)
Comment (DavidS) [inline .travis.yml]: puppetlabs_spec_helper 1.2 defaults to strict variable testing for puppet 4. (it's still disableable)

File: .rubocop.yml (hunk 2 above)
Comment (DavidS) [inline .rubocop.yml]: Ah, I was not aware of that one (and I like it more [on the surface]). The version here is copied from one of our modules, and is only meant as a starting point. we're currently in no position to activate rubocop testing on our modules anyways. I'll make a note on the appropriate ticket. Thanks for your scrutiny :-)

File: .travis.yml (hunk 1 above)
Comment (ghoneycutt) [inline .travis.yml]: Ah ha! Thanks for pointing that out.

File: .rubocop.yml (hunk 2 above)
Comment (ghoneycutt) [inline .rubocop.yml]: Thanks for the info.

I only have it enabled on a couple of modules and am still not sure on all of the cops that I want to enforce.

https://github.com/ghoneycutt/puppet-module-nscd

Comment (DavidS) [inline .rubocop.yml]: Yeah, that will be an interesting discussion. In the end I don't really care about the list, but am more interested in getting style discussions out of the way. Machines are way better at enforcing consistency....

**Comment by DavidS** [comment]
From https://github.com/puppetlabs/modulesync_configs/pull/90

**Pull request merged**